README.md

```diff
@@ -1,24 +1,89 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+## Description
 
-* Ruby version
+In this project, I created a budget website in which users can track their finances and what they're being spent on. 
 
-* System dependencies
+## Screenshots
 
-* Configuration
+![image](https://user-images.githubusercontent.com/74814780/159991267-1dcae1b2-f4a4-4837-be7a-dc4877926c5b.png)
 
-* Database creation
+![image](https://user-images.githubusercontent.com/74814780/159991298-b0889634-e711-4676-89e0-fc7f0d271367.png)
 
-* Database initialization
+[Presentation Video](https://www.loom.com/share/28511a885c3f48cea466eecce85126f5)
 
-* How to run the test suite
+## Built with
 
-* Services (job queues, cache servers, search engines, etc.)
+- Rails
 
-* Deployment instructions
+- HTML
 
-* ...
+- CSS
+
+- Linters(Rubocop and Stylelint)
+
+## Live Link
+[Demo link](http://127.0.0.1:3000/users/sign_up)
+
```

Comment:
- [ ] I would recommend deploying the application using Heroku, Netlify, or GitHub Pages.
- [ ] A link to the deployed project should be present in the README.